ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1.admin;
+
+option java_outer_classname = "UserManagementServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1.admin";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
+
+
+
+// --> See USER_MANAGEMENT_DESIGN.md for status and notes <-- //
+
+
+
+// Manage users and their rights for interacting with the Ledger API served by participant node.
+//
+// The API roughly follows the Google API style guidelines for resource-oriented design
+// (https://cloud.google.com/apis/design/resources) with the following simplifications to 
+// ease its implementation:
+// 
+// 1. List methods do not (yet) support pagination, 
+//    as we expect to have fewer than 10k users and 1k rights per user.
```

Comment:
I largely agree with these numbers, but given https://github.com/digital-asset/daml/issues/10283 this could be a tough API to change later…

Reply:
My aim would be to add pagination support as per https://cloud.google.com/apis/design/design_patterns#list_pagination

This can be done in a backwards compatible way, by assuming a default limit in case none is specified.